assets/src/edit-story/components/canvas/singleSelectionMoveable/useDrag.js

```diff
@@ -87,12 +87,15 @@ function useSingleSelectionDrag({
       setDraggingResource(null);
       return undefined;
     }
+
+    const roundToZero = (num) => (Math.abs(num) <= 1 ? 0 : num);
```

Comment:
This name is perhaps a bit confusing -- not sure of a better one either though.

When does this happen -- does snapping to the design space cause numbers that are not round?

Reply:
Sometimes it's 1, sometimes it's -1. It's because we do a lot of rounding and it's often off. It's still off - try snapping to the 48px design guideline - often the x value of the "snapped" element will be 49.

It's an underlying bug. We should fix it by doing something like this:

* If you drop an object while it's snapping to something, use the coordinate of what it's snapping to rather than the regular "initial + delta" calculation.

Reply:
See this, which also occurs in main:

![snap-rounding-off](https://user-images.githubusercontent.com/637548/94493388-75648980-01ba-11eb-9987-1fc3a973c9f7.gif)

Even though the dropped element seems to snap to the left side of an element at 100px, the dropped element x can easily vary by 1px. And actually does very often - more than not!

Reply:
Hmm, okay. Maybe let's just create a follow-up issue for now?

Reply:
Added in #4813.